src/open_inwoner/accounts/models.py

```diff
@@ -316,6 +356,7 @@ def get_contact_type_display(self) -> str:
         choice = ContactTypeChoices.get_choice(self.contact_type)
         return choice.label
 
+    @property
     def is_not_active(self):
         return not self.is_active
```

Comment:
This function is weird, why does it exist instead of `not user.is_active` like normal python?